scss/civicrm/common/_datepickers.scss

```diff
@@ -55,7 +55,8 @@
     }
 
     select {
-      @extend .form-control, %open-sans-font-family;
+      @extend .form-control;
+      font-family: 'Open Sans';
```

Comment:
Why you don't the mixin here instead? Is there a specific reason for that?

Reply:
Good point! Its better to use  the `mixin`.